src/Data/Managers/V2/helpers/useFetchHelpers.test.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { renderHook } from "@testing-library/react";
+import { createCookie, removeCookie } from "@/Data/Common/CookieHelper";
+import { dependencies } from "@/Test";
+import { DependencyProvider } from "@/UI";
+import { useFetchHelpers } from "./";
+
+const setup = (getToken: () => string | undefined = () => undefined) => {
+  const wrapper = ({ children }) => (
+    <DependencyProvider
+      dependencies={{
+        ...dependencies,
+        authHelper: {
+          ...dependencies.authHelper,
+          getToken,
+        },
+      }}
+    >
+      {children}
+    </DependencyProvider>
+  );
+  return wrapper;
+};
+
+describe("createHeaders", () => {
+  it("should return headers with environment when env is defined", () => {
+    const env = "1234abcd";
+    const wrapper = setup();
+
+    const { result } = renderHook(() => useFetchHelpers().createHeaders(env), {
+      wrapper,
+    });
+    expect(result.current.get("X-Inmanta-Tid")).toEqual(env);
+  });
+
+  it("should return headers without environment when env is undefined", () => {
+    const wrapper = setup();
+
+    const { result } = renderHook(() => useFetchHelpers().createHeaders(), {
+      wrapper,
+    });
+    expect(result.current.get("X-Inmanta-Tid")).toEqual(null);
+  });
+
+  it("should return headers without Authorization Token when authController is disabled", () => {
+    const wrapper = setup();
+
+    const { result } = renderHook(() => useFetchHelpers().createHeaders(), {
+      wrapper,
+    });
+    expect(result.current.get("Authorization")).toEqual(null);
+  });
+
+  it("should return headers without Authorization Token when authHelper hook return undefined", () => {
```

Comment:
when there is no authentication, we don't set authorization token so the value will be undefined

Reply:
getToken should return null instead of undefined, like we do everywhere else when instantiating a function like that. You do it for getUser, why make it different for getToken?